client/my-sites/backup/backup-contents-page/file-browser/file-browser-header.tsx

```diff
@@ -59,57 +46,33 @@ const FileBrowserHeader: FunctionComponent< FileBrowserHeaderProps > = ( {
 
 	return (
 		<div className="file-browser-header">
-			{ ! showCheckboxes && (
-				<div className="file-browser-header__select">
-					<Button className="file-browser-header__select-button" onClick={ onSelectClick } compact>
-						{ translate( 'Select' ) }
-					</Button>
-					<div className="file-browser-header__select-info">
-						{ translate( 'Select individual files to restore or download' ) }
-					</div>
-				</div>
-			) }
-			{ showCheckboxes && (
-				<div className="file-browser-header__selecting">
-					<FormCheckbox
-						checked={
-							rootNode
-								? rootNode.checkState === 'checked' || rootNode.checkState === 'mixed'
-								: false
-						}
-						onChange={ onCheckboxChange }
-						className={ `${ rootNode && rootNode.checkState === 'mixed' ? 'mixed' : '' }` }
-					/>
-					<div className="file-browser-header__selecting-info">
-						{ browserCheckList.totalItems } { translate( 'files selected' ) }
-					</div>
-					<Button
-						className="file-browser-header__download-button"
-						onClick={ onDownloadClick }
-						compact
-						disabled={ browserCheckList.totalItems === 0 }
-					>
-						{ translate( 'Download files' ) }
+			{ browserCheckList.totalItems > 0 && (
+				<div className="file-browser-header__action-buttons">
+					<Button className="file-browser-header__download-button" onClick={ onDownloadClick }>
+						{ translate( 'Download selected files' ) }
 					</Button>
 					<Button
 						className="file-browser-header__restore-button"
 						onClick={ onRestoreClick }
+						disabled={ isRestoreDisabled }
```

Comment:
We might want to hide the restore button if the site is a multisite, similar to [ActionButtons behavior](https://github.com/Automattic/wp-calypso/blob/5a881f0ddd815e1346e55f171572a8fd93833160/client/components/jetpack/daily-backup-status/action-buttons.jsx#L109).
This also applies to the single restore button. We can address that in a separate PR.